.github/CONTRIBUTING.md

```diff
@@ -165,8 +165,12 @@ first line in the commit message.
 please summarize important information such as why the proposed
 approach was chosen or a brief description of the bug you are resolving.
 Each line of the body must be 72 characters or less.
-* The last line must be a `Signed-off-by:` line with the developer's
-name followed by their email.
+* The last line must be a `Signed-off-by:` tag with the developer's
+name followed by their email. This is the developer's certification
+that they have the right to submit the patch for inclusion into
+the code base and indicates agreement to the [Developer's Certificate
+of Origin](https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin).
+Code without a proper signoff cannot be merged.
```

Comment:
the whole file has line breaks like this already, check the source and click 'view' to render, looks fine.